### PR TITLE
kernel-install.eclass: Compress kernel modules in parallel

### PR DESCRIPTION
**Pull request by mgorny**
Closes: https://bugs.gentoo.org/922185

### CODE REVIEW

File: eclass/kernel-install.eclass [hunk 1]
```diff
@@ -757,8 +757,10 @@ kernel-install_compress_modules() {
 	if use modules-compress; then
 		einfo "Compressing kernel modules ..."
 		# taken from scripts/Makefile.modinst
-		find "${ED}/lib" -name '*.ko' -exec \
-			xz --check=crc32 --lzma2=dict=1MiB {} + || die
+		find "${ED}/lib" -name '*.ko' -print0 |
+			xargs -0 -P "$(makeopts_jobs)" -n 128 \
+				xz --check=crc32 --lzma2=dict=1MiB
```
Comment (AndrewAmmerlaan) [inline eclass/kernel-install.eclass:762]: `linux-mod-r1.eclass` does:

```
	compress=(
			xz -q
			--memlimit-compress=50%
			--threads="$(makeopts_jobs)"
			# match options from kernel's Makefile.modinst (bug #920837)
			--check=crc32
			--lzma2=dict=1MiB
		)
```

Could we do the same here (i.e. use `--threads` and maybe `--memlimit-compress` as well).

Comment (mgorny) [reply eclass/kernel-install.eclass:762]: `--threads` is useless, as it controls compression for a single file. We have lots of small files, not one huge file.

Comment (AndrewAmmerlaan) [reply eclass/kernel-install.eclass:762]: > `--threads` is useless, as it controls compression for a single file. We have lots of small files, not one huge file.

Right, that makes sense. LGTM then.

Comment (ionenwks) [reply eclass/kernel-install.eclass:762]: Now now, it's useful when it compresses the 61MB nvidia.ko :eyes:

**Review by ionenwks — APPROVED** [review]

**Review by thesamesam — APPROVED** [review]
This could do with a comment noting why we don't `-Tn`, but LGTM. Thanks!

File: eclass/kernel-install.eclass (hunk 1 above)
Comment (mgorny) [reply eclass/kernel-install.eclass:762]: I know. I was referring to the specific case of kernel ebuild.

**Pull request closed without merging**